modules/microshift-gitops-adding-apps.adoc

```diff
@@ -62,9 +62,9 @@ spec:
 +
 [source,terminal]
 ----
-$ oc apply -f <my-app>.yaml <1>
+$ oc apply -f <my_app>.yaml <1>
```

Comment:
@ShaunaDiaz I have made the suggested change.  Earlier hyphen was used and not underscore in lines 65 and 67. should I check with engineering on this change or docs can do this change?

Reply:
The best way to go here is _<my_app.yaml>_ because it's less confusing. I would repeat that formatting on line 67 and everywhere else it appears. So `_<my_app.yaml>_` on line 67.

Reply:
done